frontend/src/lib/components/PropertiesTimeline/propertiesTimelineLogic.ts

```diff
@@ -60,12 +60,24 @@ export const propertiesTimelineLogic = kea<propertiesTimelineLogicType>([
             {
                 loadResult: async () => {
                     if (props.actor.type === 'person') {
-                        const response = (await api.get(
+                        const queryId = uuid()
+                        return await apiGetWithTimeToSeeDataTracking(
                             `api/projects/${values.currentTeamId}/persons/${
                                 props.actor.uuid
-                            }/properties_timeline/?${toParams(props.filter)}`
-                        )) as RawPropertiesTimelineResult
-                        return response
+                            }/properties_timeline/?${toParams(props.filter)}`,
+                            values.currentTeamId,
+                            {
+                                type: 'insight_load',
+                                context: 'insight',
```

Comment:
Both the `type` and `context` here overlap with actual insights with no way to distinguish between the two.

They should be something else, but I'm not sure what the right values are right now.

I don't really care about properties timeline _yet_ from a performance standpoint - it's nested too deep to matter. Should we delay this PR until we start to care?

Reply:
> type would be properties_timeline_load, context = actors_modal?

This sounds correct. :)